bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/models/form/Fragment.java

```diff
@@ -0,0 +1,46 @@
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ ~ Copyright 2023 Adobe
+ ~
+ ~ Licensed under the Apache License, Version 2.0 (the "License");
+ ~ you may not use this file except in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~     http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing, software
+ ~ distributed under the License is distributed on an "AS IS" BASIS,
+ ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+package com.adobe.cq.forms.core.components.models.form;
+
+import java.util.List;
+
+import org.apache.sling.api.resource.Resource;
+import org.osgi.annotation.versioning.ConsumerType;
+
+/**
+ * Defines the form {@code Fragment} Sling Model used for the {@code /apps/core/fd/components/form/fragment/v1/fragment} component.
+ *
+ * @since com.adobe.cq.forms.core.components.models.form 4.4.0
+ */
+@ConsumerType
+public interface Fragment extends Panel {
+
+    /**
+     * Fragment reference
+     *
+     * @return {@code fragRef}
+     * @since com.adobe.cq.forms.core.components.models.form 4.4.0
+     */
+    String getFragmentPath();
```

Comment:
Please fix the name of the API as mentioned earlier. All our API's follow the same naming.

Reply:
@rismehta as mentioned earlier it improves readability.
Also content fragment is also following same:

https://github.com/adobe/aem-core-wcm-components/blob/main/content/src/content/jcr_root/apps/core/wcm/components/contentfragment/v1/contentfragment/contentfragment.html

https://github.com/search?q=repo%3Aadobe%2Faem-core-forms-components%20sdk&type=code

Reply:
We need to keep consistent naming of the API, else it would create a confusion.

Reply:
In the link you shared, they are directly accessing the properties, hence fragmentPath is used.

Reply:
No, because getPath represents path of resource. Which is different from fragmentPath, which represents a reference to a different asset. So it can't be getPath.
cc: @rismehta